src/components/vuestic-components/va-icon/VaIcon.vue

```diff
@@ -85,7 +89,7 @@ export default {
       }
     },
     computedContent () {
-      return this.icon && this.icon.content
+      return this.icon ? this.icon.content : this.spin ? 'refresh' : ''
```

Comment:
Spin prop should function similarly to `fa-spin`. It shouldn't modify existing icon.

Reply:
Material icons working by adding text in `i` tag, not CSS classes